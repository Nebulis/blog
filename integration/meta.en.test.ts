import fs from "fs"
import path from "path"
import { parse } from "node-html-parser"

import glob from "glob"

describe("english pages", () => {
  const files: string[] = glob.sync("public/en/**/*.html")
  console.log(`Found ${files.length} english pages`)
  it.each(files)("exports meta for %s correctly", (filePath) => {
    const file = fs.readFileSync(path.resolve(filePath), { encoding: "utf8" })
    const root = parse(file)
    const description = root.querySelector('meta[name="twitter:description"]')
    expect(description?.getAttribute("content")).toMatchSnapshot("social-network")
    const title = root.querySelector("title")
    expect(title?.text).toMatchSnapshot("title")
    const googleDescription = root.querySelector('meta[name="description"]')
    expect(googleDescription?.getAttribute("content")).toMatchSnapshot("meta-description")
  })
})
