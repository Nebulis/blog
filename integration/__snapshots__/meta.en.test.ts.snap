// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`english pages exports meta for public/en/404.html correctly 1`] = `"Embark with us in this crazy adventure, a blog that reveals all our travels, and much more!"`;

exports[`english pages exports meta for public/en/404.html correctly 2`] = `"404: Not found | Travel Blog"`;

exports[`english pages exports meta for public/en/404.html correctly 3`] = `"Embark with us in this crazy adventure, a blog that reveals all our travels, and much more!"`;

exports[`english pages exports meta for public/en/404/index.html correctly 1`] = `"Embark with us in this crazy adventure, a blog that reveals all our travels, and much more!"`;

exports[`english pages exports meta for public/en/404/index.html correctly 2`] = `"404: Not found | Travel Blog"`;

exports[`english pages exports meta for public/en/404/index.html correctly 3`] = `"Embark with us in this crazy adventure, a blog that reveals all our travels, and much more!"`;

exports[`english pages exports meta for public/en/about/contact/index.html correctly 1`] = `"Do you have questions? Advice to share? Or just few words for us? Feel free to send us a message, it will be our pleasure to talk with you. ❤️"`;

exports[`english pages exports meta for public/en/about/contact/index.html correctly 2`] = `"Contact | Travel Blog"`;

exports[`english pages exports meta for public/en/about/contact/index.html correctly 3`] = `"Do you have questions? Advice to share? Or just few words for us? Feel free to send us a message, it will be our pleasure to talk with you. ❤️"`;

exports[`english pages exports meta for public/en/about/who/index.html correctly 1`] = `"After 6 years of love, daily grind and need for change of scenery, we decided in 2016 to change our life and settle in Singapore with our 2 cats without ever having set foot there."`;

exports[`english pages exports meta for public/en/about/who/index.html correctly 2`] = `"About us? | Travel Blog"`;

exports[`english pages exports meta for public/en/about/who/index.html correctly 3`] = `"A sort of diary that we share with you, a few tips here and there but above all an unforgettable memory book that we will read ourselves in the future."`;

exports[`english pages exports meta for public/en/articles/index.html correctly 1`] = `"Embark with us in this crazy adventure, a blog that reveals all our travels, and much more!"`;

exports[`english pages exports meta for public/en/articles/index.html correctly 2`] = `"main | Travel Blog"`;

exports[`english pages exports meta for public/en/articles/index.html correctly 3`] = `"Embark with us in this crazy adventure, a blog that reveals all our travels, and much more!"`;

exports[`english pages exports meta for public/en/asia/index.html correctly 1`] = `"Rest on wonderful beaches, visit of traditional temples, trek in the jungle or simply walk to discover breathtaking landscapes, from big cities to small villages, discovering the traditions of some tribes or observing silently wild animals... So many countries calling for evasion and making you dream."`;

exports[`english pages exports meta for public/en/asia/index.html correctly 2`] = `"Asia | Travel Blog"`;

exports[`english pages exports meta for public/en/asia/index.html correctly 3`] = `"Find below the most beautiful wonders of Asia as well as when and where to travel for your dream holidays. Currently you will find articles about Vietnam and the Philippines."`;

exports[`english pages exports meta for public/en/asia/japan/fuji/index.html correctly 1`] = `"Embark with us in this crazy adventure, a blog that reveals all our travels, and much more!"`;

exports[`english pages exports meta for public/en/asia/japan/fuji/index.html correctly 2`] = `"japan | Travel Blog"`;

exports[`english pages exports meta for public/en/asia/japan/fuji/index.html correctly 3`] = `"Embark with us in this crazy adventure, a blog that reveals all our travels, and much more!"`;

exports[`english pages exports meta for public/en/asia/japan/fuji/lake-kawaguchiko/index.html correctly 1`] = `"Embark with us in this crazy adventure, a blog that reveals all our travels, and much more!"`;

exports[`english pages exports meta for public/en/asia/japan/fuji/lake-kawaguchiko/index.html correctly 2`] = `"main | Travel Blog"`;

exports[`english pages exports meta for public/en/asia/japan/fuji/lake-kawaguchiko/index.html correctly 3`] = `"Embark with us in this crazy adventure, a blog that reveals all our travels, and much more!"`;

exports[`english pages exports meta for public/en/asia/japan/himeji/castle/index.html correctly 1`] = `"Embark with us in this crazy adventure, a blog that reveals all our travels, and much more!"`;

exports[`english pages exports meta for public/en/asia/japan/himeji/castle/index.html correctly 2`] = `"main | Travel Blog"`;

exports[`english pages exports meta for public/en/asia/japan/himeji/castle/index.html correctly 3`] = `"Embark with us in this crazy adventure, a blog that reveals all our travels, and much more!"`;

exports[`english pages exports meta for public/en/asia/japan/himeji/index.html correctly 1`] = `"Embark with us in this crazy adventure, a blog that reveals all our travels, and much more!"`;

exports[`english pages exports meta for public/en/asia/japan/himeji/index.html correctly 2`] = `"japan | Travel Blog"`;

exports[`english pages exports meta for public/en/asia/japan/himeji/index.html correctly 3`] = `"Embark with us in this crazy adventure, a blog that reveals all our travels, and much more!"`;

exports[`english pages exports meta for public/en/asia/japan/index.html correctly 1`] = `"Embark with us in this crazy adventure, a blog that reveals all our travels, and much more!"`;

exports[`english pages exports meta for public/en/asia/japan/index.html correctly 2`] = `"Japon | Travel Blog"`;

exports[`english pages exports meta for public/en/asia/japan/index.html correctly 3`] = `"Embark with us in this crazy adventure, a blog that reveals all our travels, and much more!"`;

exports[`english pages exports meta for public/en/asia/japan/kyoto/arashiyama/index.html correctly 1`] = `"Embark with us in this crazy adventure, a blog that reveals all our travels, and much more!"`;

exports[`english pages exports meta for public/en/asia/japan/kyoto/arashiyama/index.html correctly 2`] = `"main | Travel Blog"`;

exports[`english pages exports meta for public/en/asia/japan/kyoto/arashiyama/index.html correctly 3`] = `"Embark with us in this crazy adventure, a blog that reveals all our travels, and much more!"`;

exports[`english pages exports meta for public/en/asia/japan/kyoto/daigoji/index.html correctly 1`] = `"Embark with us in this crazy adventure, a blog that reveals all our travels, and much more!"`;

exports[`english pages exports meta for public/en/asia/japan/kyoto/daigoji/index.html correctly 2`] = `"main | Travel Blog"`;

exports[`english pages exports meta for public/en/asia/japan/kyoto/daigoji/index.html correctly 3`] = `"Embark with us in this crazy adventure, a blog that reveals all our travels, and much more!"`;

exports[`english pages exports meta for public/en/asia/japan/kyoto/fushimi-inari-taisha/index.html correctly 1`] = `"Embark with us in this crazy adventure, a blog that reveals all our travels, and much more!"`;

exports[`english pages exports meta for public/en/asia/japan/kyoto/fushimi-inari-taisha/index.html correctly 2`] = `"main | Travel Blog"`;

exports[`english pages exports meta for public/en/asia/japan/kyoto/fushimi-inari-taisha/index.html correctly 3`] = `"Embark with us in this crazy adventure, a blog that reveals all our travels, and much more!"`;

exports[`english pages exports meta for public/en/asia/japan/kyoto/index.html correctly 1`] = `"Embark with us in this crazy adventure, a blog that reveals all our travels, and much more!"`;

exports[`english pages exports meta for public/en/asia/japan/kyoto/index.html correctly 2`] = `"japan | Travel Blog"`;

exports[`english pages exports meta for public/en/asia/japan/kyoto/index.html correctly 3`] = `"Embark with us in this crazy adventure, a blog that reveals all our travels, and much more!"`;

exports[`english pages exports meta for public/en/asia/japan/kyoto/kinkakuji/index.html correctly 1`] = `"Embark with us in this crazy adventure, a blog that reveals all our travels, and much more!"`;

exports[`english pages exports meta for public/en/asia/japan/kyoto/kinkakuji/index.html correctly 2`] = `"main | Travel Blog"`;

exports[`english pages exports meta for public/en/asia/japan/kyoto/kinkakuji/index.html correctly 3`] = `"Embark with us in this crazy adventure, a blog that reveals all our travels, and much more!"`;

exports[`english pages exports meta for public/en/asia/japan/nagoya/castle/index.html correctly 1`] = `"Embark with us in this crazy adventure, a blog that reveals all our travels, and much more!"`;

exports[`english pages exports meta for public/en/asia/japan/nagoya/castle/index.html correctly 2`] = `"main | Travel Blog"`;

exports[`english pages exports meta for public/en/asia/japan/nagoya/castle/index.html correctly 3`] = `"Embark with us in this crazy adventure, a blog that reveals all our travels, and much more!"`;

exports[`english pages exports meta for public/en/asia/japan/nagoya/index.html correctly 1`] = `"Embark with us in this crazy adventure, a blog that reveals all our travels, and much more!"`;

exports[`english pages exports meta for public/en/asia/japan/nagoya/index.html correctly 2`] = `"japan | Travel Blog"`;

exports[`english pages exports meta for public/en/asia/japan/nagoya/index.html correctly 3`] = `"Embark with us in this crazy adventure, a blog that reveals all our travels, and much more!"`;

exports[`english pages exports meta for public/en/asia/japan/nagoya/toganji/index.html correctly 1`] = `"Embark with us in this crazy adventure, a blog that reveals all our travels, and much more!"`;

exports[`english pages exports meta for public/en/asia/japan/nagoya/toganji/index.html correctly 2`] = `"main | Travel Blog"`;

exports[`english pages exports meta for public/en/asia/japan/nagoya/toganji/index.html correctly 3`] = `"Embark with us in this crazy adventure, a blog that reveals all our travels, and much more!"`;

exports[`english pages exports meta for public/en/asia/japan/spring-in-japan/index.html correctly 1`] = `"Embark with us in this crazy adventure, a blog that reveals all our travels, and much more!"`;

exports[`english pages exports meta for public/en/asia/japan/spring-in-japan/index.html correctly 2`] = `"Le Japon au printemps | Travel Blog"`;

exports[`english pages exports meta for public/en/asia/japan/spring-in-japan/index.html correctly 3`] = `"Embark with us in this crazy adventure, a blog that reveals all our travels, and much more!"`;

exports[`english pages exports meta for public/en/asia/japan/tokyo/a-day-in-tokyo/index.html correctly 1`] = `"Embark with us in this crazy adventure, a blog that reveals all our travels, and much more!"`;

exports[`english pages exports meta for public/en/asia/japan/tokyo/a-day-in-tokyo/index.html correctly 2`] = `"main | Travel Blog"`;

exports[`english pages exports meta for public/en/asia/japan/tokyo/a-day-in-tokyo/index.html correctly 3`] = `"Embark with us in this crazy adventure, a blog that reveals all our travels, and much more!"`;

exports[`english pages exports meta for public/en/asia/japan/tokyo/hamarikyu-garden/index.html correctly 1`] = `"Embark with us in this crazy adventure, a blog that reveals all our travels, and much more!"`;

exports[`english pages exports meta for public/en/asia/japan/tokyo/hamarikyu-garden/index.html correctly 2`] = `"main | Travel Blog"`;

exports[`english pages exports meta for public/en/asia/japan/tokyo/hamarikyu-garden/index.html correctly 3`] = `"Embark with us in this crazy adventure, a blog that reveals all our travels, and much more!"`;

exports[`english pages exports meta for public/en/asia/japan/tokyo/index.html correctly 1`] = `"Embark with us in this crazy adventure, a blog that reveals all our travels, and much more!"`;

exports[`english pages exports meta for public/en/asia/japan/tokyo/index.html correctly 2`] = `"japan | Travel Blog"`;

exports[`english pages exports meta for public/en/asia/japan/tokyo/index.html correctly 3`] = `"Embark with us in this crazy adventure, a blog that reveals all our travels, and much more!"`;

exports[`english pages exports meta for public/en/asia/japan/tokyo/sensoji/index.html correctly 1`] = `"Embark with us in this crazy adventure, a blog that reveals all our travels, and much more!"`;

exports[`english pages exports meta for public/en/asia/japan/tokyo/sensoji/index.html correctly 2`] = `"main | Travel Blog"`;

exports[`english pages exports meta for public/en/asia/japan/tokyo/sensoji/index.html correctly 3`] = `"Embark with us in this crazy adventure, a blog that reveals all our travels, and much more!"`;

exports[`english pages exports meta for public/en/asia/japan/transports/index.html correctly 1`] = `"Embark with us in this crazy adventure, a blog that reveals all our travels, and much more!"`;

exports[`english pages exports meta for public/en/asia/japan/transports/index.html correctly 2`] = `"main | Travel Blog"`;

exports[`english pages exports meta for public/en/asia/japan/transports/index.html correctly 3`] = `"Embark with us in this crazy adventure, a blog that reveals all our travels, and much more!"`;

exports[`english pages exports meta for public/en/asia/philippines/el-nido/el-nido-paradise/index.html correctly 1`] = `"Breathtaking landscapes, white sands and turquoise water ... El Nido must imperatively be part of your dream travel list for this new year."`;

exports[`english pages exports meta for public/en/asia/philippines/el-nido/el-nido-paradise/index.html correctly 2`] = `"El Nido: A little paradise | Travel Blog"`;

exports[`english pages exports meta for public/en/asia/philippines/el-nido/el-nido-paradise/index.html correctly 3`] = `"In this article we tell you when to go? How many days ? What budget to plan? How to get there ? And where to stay? We give you advice based on our own experience and tell you about our adventure on the island of Palawan."`;

exports[`english pages exports meta for public/en/asia/philippines/el-nido/index.html correctly 1`] = `"Do you want to join us to discover the jewel of the Philippines on the Palawan Island? Grab your mask and snorkel or dive in these crystal clear waters that will leave you a lasting impression. Take advantage of the different tours called \\"Island hopping\\" to choose what suits you best."`;

exports[`english pages exports meta for public/en/asia/philippines/el-nido/index.html correctly 2`] = `"El Nido | Travel Blog"`;

exports[`english pages exports meta for public/en/asia/philippines/el-nido/index.html correctly 3`] = `"In each article, we answer to your questions: When to go? How many days ? What budget to plan? For each place visited, we try to convey our visit and our feelings as well as possible. We also tell you How to get there? Why ? When? How long and at what period? Without forgetting the addresses and prices."`;

exports[`english pages exports meta for public/en/asia/philippines/index.html correctly 1`] = `"There is no doubt this is the country that made us travel and dream the most, thanks to its breathtaking landscapes, its unique gorgeous islands and its immense snorkeling and diving spots. The Philippines comprises more than 7,600 islands, which means that you are spoilt for choice for your dream vacation. Enjoy discover through our articles and our photos, our trip to the Philippines."`;

exports[`english pages exports meta for public/en/asia/philippines/index.html correctly 2`] = `"Philippines | Travel Blog"`;

exports[`english pages exports meta for public/en/asia/philippines/index.html correctly 3`] = `"There is no doubt this is the country that made us travel and dream the most, thanks to its breathtaking landscapes, its unique gorgeous islands and its immense snorkeling and diving spots. The Philippines comprises more than 7,600 islands, which means that you are spoilt for choice for your dream vacation. Enjoy discover through our articles and our photos, our trip to the Philippines."`;

exports[`english pages exports meta for public/en/asia/vietnam/index.html correctly 1`] = `"Vietnam is a unique destination. Magnificent and varied landscapes, natural parks, a cultural heritage and a rich historical heritage ... From Ho Chi Minh City to Hanoi, via Da Nang, each destination offers its own history and a different experience. Discover through our articles and our photos our itinerary in the Southern Vietnam. We will share very soon about the North and the Center so stay tuned."`;

exports[`english pages exports meta for public/en/asia/vietnam/index.html correctly 2`] = `"Vietnam | Travel Blog"`;

exports[`english pages exports meta for public/en/asia/vietnam/index.html correctly 3`] = `"Vietnam is a unique destination. Magnificent and varied landscapes, natural parks, a cultural heritage and a rich historical heritage ... From Ho Chi Minh City to Hanoi, via Da Nang, each destination offers its own history and a different experience. Discover through our articles and our photos our itinerary in the Southern Vietnam. We will share very soon about the North and the Center so stay tuned."`;

exports[`english pages exports meta for public/en/asia/vietnam/southern-vietnam/can-tho/index.html correctly 1`] = `"Can Tho is the largest city in the Mekong River Delta, located on the banks of the Hau River, it is renowned for its very large floating market."`;

exports[`english pages exports meta for public/en/asia/vietnam/southern-vietnam/can-tho/index.html correctly 2`] = `"Cân Thơ floating market | Travel Blog"`;

exports[`english pages exports meta for public/en/asia/vietnam/southern-vietnam/can-tho/index.html correctly 3`] = `"Our trip to Can Tho and more particularly in the Cai Rang Floating Market, and in addition, a visit to a rice paper factory. In this article we also give you advice on how to get there? How many days ? When to go? And where to stay?"`;

exports[`english pages exports meta for public/en/asia/vietnam/southern-vietnam/cu-chi-tunnels/index.html correctly 1`] = `"Huge network of underground tunnels reaching about 250 km, from Ho Chi Minh City to the border of Cambodia."`;

exports[`english pages exports meta for public/en/asia/vietnam/southern-vietnam/cu-chi-tunnels/index.html correctly 2`] = `"Củ Chi tunnels | Travel Blog"`;

exports[`english pages exports meta for public/en/asia/vietnam/southern-vietnam/cu-chi-tunnels/index.html correctly 3`] = `"With this article we try to explain to you what the Cu Chi tunnels are. We take you with us to visit them and discover the history of Vietnam. We answer all your questions: How to get there? Why ? When? How long and at what period? We also tell you the price and where to stay."`;

exports[`english pages exports meta for public/en/asia/vietnam/southern-vietnam/discover/index.html correctly 1`] = `"Southern Vietnam is full of surprises and discoveries, each one more exciting. From dynamic cities like Hô Chi Minh City, to boat trips on the Mekong River Delta under palm trees, there is something for everyone."`;

exports[`english pages exports meta for public/en/asia/vietnam/southern-vietnam/discover/index.html correctly 2`] = `"Discover Southern Vietnam | Travel Blog"`;

exports[`english pages exports meta for public/en/asia/vietnam/southern-vietnam/discover/index.html correctly 3`] = `"When to go? How many days ? What budget to plan? Questions that we often ask ourselves before travelling. We answer to your questions based on our own experience. We also show you our travel itinerary for 1 week in Southern Vietnam."`;

exports[`english pages exports meta for public/en/asia/vietnam/southern-vietnam/index.html correctly 1`] = `"Travel with us in Southern Vietnam through our travelogue. Discover big cities like Ho Chi Minh City, its museum and tunnels, take off to magnificent landscapes around the Mekong Delta River on islands like Ben Tre or visit the Cân Thơ floating market with us."`;

exports[`english pages exports meta for public/en/asia/vietnam/southern-vietnam/index.html correctly 2`] = `"Southern Vietnam | Travel Blog"`;

exports[`english pages exports meta for public/en/asia/vietnam/southern-vietnam/index.html correctly 3`] = `"In each article, we answer to your questions: When to go? How many days ? What budget to plan? For each place visited, we try to convey our visit and our feelings as well as possible. We also tell you How to get there? Why ? When? How long and at what period? Without forgetting the addresses and prices."`;

exports[`english pages exports meta for public/en/asia/vietnam/southern-vietnam/my-tho/index.html correctly 1`] = `"Away from the hustle and bustle of Ho Chi Minh City, discover the town of Mỹ Tho and take a boat trip to Ben Tre to discover the huge tropical gardens that surround you."`;

exports[`english pages exports meta for public/en/asia/vietnam/southern-vietnam/my-tho/index.html correctly 2`] = `"Cruise on the Mekong between Mỹ Tho and Ben Tre | Travel Blog"`;

exports[`english pages exports meta for public/en/asia/vietnam/southern-vietnam/my-tho/index.html correctly 3`] = `"Ready for a boat trip in Vietnam? In this article, we share our opinion on our visit from Mỹ Tho to Ben Tre, and ideas about what to do on the Mekong River Delta. We also give you advice on how to get there? How many days ? When to go? And where to stay?"`;

exports[`english pages exports meta for public/en/asia/vietnam/southern-vietnam/war-remnants-museum/index.html correctly 1`] = `"The War Remnants Museum bears witness to the atrocities of the Indochina War and the Vietnam War and reveals us the ongoing impact on Vietnamese lives."`;

exports[`english pages exports meta for public/en/asia/vietnam/southern-vietnam/war-remnants-museum/index.html correctly 2`] = `"War Remnants Museum in Ho Chi Minh City | Travel Blog"`;

exports[`english pages exports meta for public/en/asia/vietnam/southern-vietnam/war-remnants-museum/index.html correctly 3`] = `"Discover the War Remnants Museum in Ho Chi Minh City (also known as Saigon) through our visit, opinion and get to know more about the VietnamWar. In this article, we also give you many information : How to get there ? Why ? When ? How long and what time of year ? We also let you know about the location and the price."`;

exports[`english pages exports meta for public/en/asia/vietnam/transports/index.html correctly 1`] = `"Embark with us in this crazy adventure, a blog that reveals all our travels, and much more!"`;

exports[`english pages exports meta for public/en/asia/vietnam/transports/index.html correctly 2`] = `"main | Travel Blog"`;

exports[`english pages exports meta for public/en/asia/vietnam/transports/index.html correctly 3`] = `"Embark with us in this crazy adventure, a blog that reveals all our travels, and much more!"`;

exports[`english pages exports meta for public/en/credits/index.html correctly 1`] = `"Embark with us in this crazy adventure, a blog that reveals all our travels, and much more!"`;

exports[`english pages exports meta for public/en/credits/index.html correctly 2`] = `"main | Travel Blog"`;

exports[`english pages exports meta for public/en/credits/index.html correctly 3`] = `"Embark with us in this crazy adventure, a blog that reveals all our travels, and much more!"`;

exports[`english pages exports meta for public/en/index.html correctly 1`] = `"Passionate about travel, we share our adventures around the world on our blog. Tips, advice, journeys, journal, anecdotes, favorites... we hope you will find what you are looking for! A travel and lifestyle blog made by a traveling couple with their heads full of dreams!"`;

exports[`english pages exports meta for public/en/index.html correctly 2`] = `"Magic of travels | Travel Blog"`;

exports[`english pages exports meta for public/en/index.html correctly 3`] = `"Passionate about travel, we share our adventures around the world on our blog. Tips, advice, journeys, journal, anecdotes, favorites... we hope you will find what you are looking for! A travel and lifestyle blog made by a traveling couple with their heads full of dreams!"`;

exports[`english pages exports meta for public/en/organisation/when-to-go/spring/index.html correctly 1`] = `"Winter is over! It's time to organize your spring holidays But where to travel during the month of March, April and May? Sunny days are back. It's time to enjoy the flowery landscapes, blue and sunny skies, and perfect outdoors temperatures. In Asia, before the summer, appreciate the last favorable months and the still bearable heat. From heavenly beaches of to jungle trek, without forgetting the cultural heritage specific to each country you have plenty of choice but hurry up!"`;

exports[`english pages exports meta for public/en/organisation/when-to-go/spring/index.html correctly 2`] = `"Travel in spring | Travel Blog"`;

exports[`english pages exports meta for public/en/organisation/when-to-go/spring/index.html correctly 3`] = `"Winter is over! It's time to organize your spring holidays But where to travel during the month of March, April and May? Currently you will find articles about Vietnam and the Philippines but new countries will be added soon."`;

exports[`english pages exports meta for public/en/organisation/when-to-go/winter/index.html correctly 1`] = `"Do you prefer snowy and magical Christmas? Or lack of sunlight make you feel low? Have holidays near astounding landscapes covered with a large white coat, rent a chalet for a cocooning week or fly for ski vacations and learn about winter sports. Or else ... Escape cold winter for heavenly beaches. Escape for the tropical sun. Discover unique landscapes and discover new cultures. So many places to spend your dream holidays during December, January or February. Alors, vous avez décidé ? Plutôt neige ou soleil en hiver ?"`;

exports[`english pages exports meta for public/en/organisation/when-to-go/winter/index.html correctly 2`] = `"Travel in winter | Travel Blog"`;

exports[`english pages exports meta for public/en/organisation/when-to-go/winter/index.html correctly 3`] = `"Do you prefer snowy and magical Christmas? Or lack of sunlight make you feel low? Currently you will find articles about Vietnam and the Philippines but new countries will be added soon.}"`;
